istioctl/cmd/istiodconfig.go

```diff
@@ -374,16 +374,19 @@ func istiodLogCmd() *cobra.Command {
 		Use:   "log [<pod-name>] [--level <scope>:<level>][--stack-trace-level <scope>:<level>]|[-r|--reset]|[--output|-o short|yaml]",
 		Short: "Manage istiod logging.",
 		Long:  "Retrieve or update logging levels of istiod components.",
-		Example: ` # Retrieve information about istiod logging levels.
-  istioctl istiod-config log [<pod-name>]
+		Example: `  # Retrieve information about istiod logging levels.
+  istioctl experimental istiod log
+
+  # Retrieve information about istiod logging levels on a specific control plane pod.
```

Comment:
Not sure if this is right, from the command it looks like that it can only get the logs of the `istiod` pod?  or is this means that this can work with multiple revisions of the control plane?

Reply:
This command can only get/set the log level of an istiod pod.

It would be better if it worked differently, for example if it took a `--revision <revision>` and used that to match for `istio.io/rev=<revision>`.  It would also be better if it set the level on ALL matching pods, not just one of them.

This work item is just about moving the command, not improving it.

Reply:
Sounds good.